public/locales/en/booklet.json

```diff
@@ -2,7 +2,7 @@
   "webpageTitle": "Booklet",
   "heading": "Beneath the Surface",
   "subtext": "30 ways to pray for Japan",
-  "subtitle": "Subtitle for an “about this book” -- Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo.",
+  "subtitle": "The Beneath the Surface prayer guide was produced in collaboration with missionaries and leaders from 17 organizations. Since its first printing in 2020, near 20,000 copies have been distributed around the world. It is available in English, Japanese, and a few other languages.The guide presents 30 priority topics for prayer in Japanese culture and society, and church and mission. They represent some of this decade's greatest challenges and opportunities for the gospel in Japan.",
```

Comment:
I think the copy changed since you last worked on this - it's now 2 paragraphs and the wording has changed a bit. I'll make a note to recheck the wording before the conference
```
"The Beneath the Surface prayer guide was produced in collaboration with missionaries and leaders from 17 Christian organizations. Since its first printing in 2020, about 20,000 copies have been distributed around the world. It is available in English, Japanese, and a few other languages.

This guide introduces 30 prayer topics for Japanese culture and society, and church and mission. The 30 topics chosen were prioritized by Japanese leaders as key opportunities and challenges for the gospel in Japan today."
```